pygmt/src/basemap.py

```diff
@@ -82,13 +82,13 @@ def basemap(self, **kwargs):
     compass : str
         Draws a map magnetic rose on the map at the location defined by the
         reference and anchor points
-    {U}
-    {V}
-    {XY}
-    {c}
-    {f}
-    {p}
-    {t}
+    {timestamp}
+    {verbose}
+    {xyshift}
+    {panel}
+    {coltypes}
+    {perspective}
+    {transparency}
```

Comment:
I like the long names, but then there's a small problem with sorting the parameters. Previously we sorted by asciibetical order (e.g. ABCabc), and it made sense since the small-letter GMT parameters like `c`/`f`/`p`/`t` were rarely used. Now that we switch to the long-form, do we still go with the original GMT sorting, or change to sorting by a long-parameter's first character?

Reply:
I don't have a good answer for this, but packages like numpy and pandas don't sort their parameters alphabetically. I think we also don't have to do that.